phones/models.py

```diff
@@ -182,7 +182,7 @@ def mark_verified(self):
         return self
 
 
-@receiver(post_save, sender=RealPhone)
+@receiver(post_save, sender=RealPhone, dispatch_uid="realphone_post_save")
```

Comment:
Even though we don't need it for this change, a `dispatch_uid` is still a good idea, so please keep this.